src/components/Sources/Sources.js

```diff
@@ -0,0 +1,66 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import React from "react";
+import { sourcesPropTypes } from "./propTypes";
+
+import "./Sources.scss";
+
+const Sources = ({ data }) => {
```

Comment:
Let's make sure to add a docstring that explains the expected output format.

Reply:
@jessex separators (`,` and `and`) replaced from js realization to css `before` and `after`. I think it became clearer now.